java/http/vertx/java-http-vertx-producer/src/main/java/HttpKafkaProducerConfig.java

```diff
@@ -45,7 +45,8 @@ public class HttpKafkaProducerConfig {
     private HttpKafkaProducerConfig(String hostname, int port, 
                                     String topic, int sendInterval,
                                     Optional<Long> messageCount,
-                                    String endpointPrefix) {
+                                    String endpointPrefix
+    ) {
```

Comment:
back to have parenthesis where they were before.

Reply:
Reverted